docker-compose.yml

```diff
@@ -65,8 +70,19 @@ services:
     ports:
       - "8080:8080"
     volumes:
-      - "${OTOPS_PATH_DEPLOYMENT_WEBAPP}:/srv/site"
-      - "${OTOPS_PATH_CONFIG_WEBAPP}/nginx.conf:/etc/nginx/conf.d/default.conf"
+      - "webapp:/srv/site"
+      - "./config/webapp/nginx.conf:/etc/nginx/conf.d/default.conf"
```

Comment:
I would have kept this with the variable, in case we'd like to map to different alternative configurations that are not the one attached to the repository